showcase/app/templates/components/app-header/index.hbs

```diff
@@ -10,6 +10,7 @@
 </Shw::Text::H1>
 
 <section data-test-percy>
+
```

Comment:
Comment: Personally, I don't like to both indent content AND include a blank line before it like this as it seems redundant so that's why I didn't have a blank line here...